etc-conf/dbus/com.redhat.RHSM1.conf

```diff
@@ -23,6 +23,12 @@
         <allow send_destination="com.redhat.RHSM1"
             send_interface="com.redhat.RHSM1"/>
 
+        <allow send_destination="com.redhat.RHSM1"
+            send_interface="com.redhat.RHSM1.Config"/>
+
+        <allow send_destination="com.redhat.RHSM1"
+            send_interface="com.redhat.RHSM1.RegisterServer"/>
```

Comment:
I found that Facts service doesn't respond to non-root (as intended). :smile:

Note to self about registration flow (and others coming back to this):
 1. `dbus-send --system --print-reply --dest=com.redhat.RHSM1 '/com/redhat/RHSM1/RegisterServer' com.redhat.RHSM1.RegisterServer.Start`
 2. `dbus-send --address=unix:abstract=${ADDRESS_FROM_ABOVE} --print-reply --dest=com.redhat.RHSM1 '/com/redhat/RHSM1/Register' 'com.redhat.RHSM1.Register.Register' string:${ORG} string:${USERNAME} string:${PASSWORD} dict:string:string:  # replace all vars appropriately`

I found that although a non-root user can get a domain socket back from RegisterServer, they cannot actually use Register service to reply, which means they can't actually register (which is okay for now, since we plan to implement non-root access later). :smile: